MT-CTF/capturetheflag
# Fix texurepacks being able to hide grenade smoke

# How to Test
* Join the game and play around with smoke grenades, make sure they still work fine
* Enable this texturepack and rejoin: [test.zip](https://github.com/MT-CTF/capturetheflag/files/6422320/test.zip)
* Make sure smoke still shows up
